src/components/Markdown/Markdown.scss

```diff
@@ -138,125 +138,100 @@
     }
   }
 
-  .table {
+  table {
```

Comment:
TBH i would avoid tables in favor of flex or css grid via divs, but guess you had reasons for this

Reply:
That is exactly the problem here. All that CSS gimmicking tables with divs when you can use `table` directly, which is exactly the use case here.

64 additions and 89 deletions in this case, advantages of using the right element.